devito/finite_differences/differentiable.py

```diff
@@ -115,6 +120,14 @@ def _eval_at(self, func):
             return self
         return self.func(*[getattr(a, '_eval_at', lambda x: a)(func) for a in self.args])
 
+    @property
+    def _eval_deriv(self):
+        return self.func(*[getattr(a, '_eval_deriv', a) for a in self.args])
+
+    @property
+    def _fd_priority(self):
```

Comment:
Would you mind quickly outlining the design decisions behind this `_fd_priority`? i.e. why it needs/was chosen to be fractional number.

Reply:
oh, forget it, just saw the resolved conversation below :p